internal/command/deploy/machines.go

```diff
@@ -563,25 +563,25 @@ func (md *machineDeployment) resolveUpdatedMachineConfig(origMachineRaw *api.Mac
 	}
 
 	launchInput.Config.Image = md.img.Tag
-	launchInput.Config.Metrics = md.appConfig.Metrics
 	launchInput.Config.Restart = origMachineRaw.Config.Restart
-	for _, s := range md.appConfig.Statics {
-		launchInput.Config.Statics = append(launchInput.Config.Statics, &api.Static{
-			GuestPath: s.GuestPath,
-			UrlPrefix: s.UrlPrefix,
-		})
-	}
-	launchInput.Config.Env = make(map[string]string)
-	for k, v := range md.appConfig.Env {
-		launchInput.Config.Env[k] = v
-	}
+	launchInput.Config.Guest = origMachineRaw.Config.Guest
+	launchInput.Config.Init = origMachineRaw.Config.Init
+	launchInput.Config.Env = lo.Assign(md.appConfig.Env)
+
 	if launchInput.Config.Env["PRIMARY_REGION"] == "" && origMachineRaw.Config.Env["PRIMARY_REGION"] != "" {
 		launchInput.Config.Env["PRIMARY_REGION"] = origMachineRaw.Config.Env["PRIMARY_REGION"]
 	}
 
+	// Anything below this point doesn't apply to machines created to run ReleaseCommand
+	if forReleaseCommand {
+		return launchInput
+	}
+
+	launchInput.Config.Metrics = md.appConfig.Metrics
+
 	if origMachineRaw.Config.Mounts != nil {
 		launchInput.Config.Mounts = origMachineRaw.Config.Mounts
-	} else if md.appConfig.Mounts != nil {
+	} else if md.appConfig.Mounts != nil && !forReleaseCommand {
```

Comment:
you shouldn't get here cause line 577 returns.

Reply:
true!

Reply:
removed by 30f80df7